docs/api/ReactWrapper/wrapProp.md

```diff
@@ -0,0 +1,60 @@
+# `.wrapProp(propName) => ReactWrapper`
+
+Returns a new wrapper around the component provided to the original wrapper's prop `propName`.`
+
+NOTE: can only be called on wrapper of a single non-DOM component element node.
```

Comment:
this doesn't seem entirely necessary to me - obviously it'd be weird to have a prop that's a react element on a DOM node, but why restrict it?

Reply:
I agree, did it like this because I thought it was a "convention". Will do the change

Reply:
Only on the methods where it makes no sense to call it on a DOM element :-)